dashboard/test/models/pd/application/application_base_test.rb

```diff
@@ -65,5 +65,18 @@ class ApplicationBaseTest < ActiveSupport::TestCase
 
       assert_equal school_info.effective_school_district_name.titleize, application.district_name
     end
+
+    test 'send_all_decision_notification_emails only sends once per application' do
+      application = create :pd_facilitator1819_application
+      application.update(status: 'declined')
+      application.lock!
+
+      mock_mail = stub
+      mock_mail.stubs(:deliver_now).returns(nil)
+      Pd::Application::Facilitator1819ApplicationMailer.expects(:declined).times(1).returns(mock_mail)
```

Comment:
note: this can also be written this way (your call):
```ruby
      Pd::Application::Facilitator1819ApplicationMailer.expects(:declined).times(1).returns(
        mock {|mail| mail.expects(:deliver_now)}
      )
```

Also note `times(1)` is default with `expects` and not needed, though of course it's fine to be more explicit.